docs/running-on-yarn.md

```diff
@@ -74,6 +74,14 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   <td>5000</td>
   <td>
     The interval in ms in which the Spark application master heartbeats into the YARN ResourceManager.
+    To avoid the application master to be expired by late reporting, if a higher value is provided, the interval will be set to the half of the expiry interval in YARN's configuration <code>(yarn.am.liveness-monitor.expiry-interval-ms / 2)</code>.
```

Comment:
Do you mind breaking these long lines into multiple lines?

I'd also rephrase this: "The value is capped at half the value of YARN's configuration for the expiry interval (<code>(yarn.am.liveness-monitor.expiry-interval-ms</code>)."